README.md

```diff
@@ -1 +1,184 @@
-# DeepStack-CameraUI
\ No newline at end of file
+# DeepStack-CameraUI
+DeepStack-CameraUI is a real-time computer vision web-application. Powered by Deepstack and Streamlit, this application performs realtime AI processing such as **object detection**, **face recognition**, **face detection** and **custom detection** on video frames from WebCam, Wired Camera or IP camera through an interactive web interface.
+
+# Installation
+
+To run DeepStack-CameraUI, you need to install the following:
+* **Python 3.8** 
+
+  - [Windows](https://www.python.org/ftp/python/3.8.0/python-3.8.0-amd64.exe) 
+  - [MacOS](https://www.python.org/ftp/python/3.8.0/python-3.8.0-macosx10.9.pkg)
+  - [Ubuntu](https://www.python.org/ftp/python/3.8.0/Python-3.8.0.tar.xz)
+
+* [Deepstack](https://docs.deepstack.cc/object-detection/index.html) 
+* Clone the reporsitory 
+* Install the dependencies by navigating into the project repository with `pip install -r requiremnets.txt`
+
+# Run DeepStack-CameraUI
+To run this streamlit application **locally** , follow these steps 
+
+* Step 1 : Run deepstack and streamlit
```

Comment:
Let the steps be like below



- Step 1 : Run DeepStack and the App

    - First run Deepstack with this command:
        ```
        docker run -e VISION-FACE=True -v localstorage:/datastore -p 80:5000 deepquestai/deepstack
        ```
Check [Deepstack Documantation](https://docs.deepstack.cc/) for more options 
    - then navigate to the src folder and run the app using the command :
        ```
        streamlit run app.py
        ```
    - Then go to your browser to load the app by visiting the link `https://localhost:8501`

- Step 2 : Configure settings and start Detection
    - Select your API
    - Enter DeepStack URL (e.g `https://localhost:80` )
    - Enter your camera address
        - For a WebCam, it will be `0` **Windows** and `-1` on **Ubuntu**
        - For IP Camera, type in the full address of video stream for the camera ( E.g `http://192.43.51.1:9091/video` )
     - Press on **Start** button